webapp/commonComponents/form/input.js

```diff
@@ -24,21 +24,24 @@ export const Input = React.forwardRef((props, ref) => {
   } = props
 
   return (
-    <TooltipError messages={validation.errors}>
+    <Tooltip
+      messages={validation.errors}
+      type={`${validation.errors ? 'error' : ''}`}
+      style={{ display: 'grid', width: '100%', height: '100%' }}>
```

Comment:
why would you need this inline style?

Reply:
Yes it doesn't look good, I'll find a better solution